wagtail/search/backends/database/sqlite/query.py

```diff
@@ -248,9 +247,7 @@ def normalize(search_query: SearchQuery) -> Tuple[SearchQuery]:
         ):  # If there are no negated subqueries, return an And(), now without the redundant MatchAll subqueries.
```

Comment:
@laymonage this file is full of trailing comments on multi-line statements that could be single line statements if the comments are moved to their own line. Do you want every instance of this changed?

Reply:
Ah, OK. I wasn't sure which bits were manually modified and which ones were auto-formatted. The changes in `wagtail/search/backends/database/mysql/query.py` seem similar, hence I made the suggestion to do it here as well. If those changes were manually made, I think it's either we make them all, or undo those changes.

(The `.format()` issue in the PR description makes sense to be manually edited I think.)

https://github.com/wagtail/wagtail/pull/11220/files#diff-72a2e9508512a738e0447971a82c79c9920871465bba681913a08616bcecd29eR237-R240

Reply:
I don't recall what happened in that specific case and it looks like there is exactly one other instance where I may have manually moved the comments to get automated one-liner formatting. I could revert those two cases to see if `ruff format` had done something odd that I reacted to, or if I just go carried away while reviewing ;) Everything else was automated aside from fixing the odd string concatenation mentioned in my original post.